pkgs/by-name/bl/blheli-configurator/package.nix

```diff
@@ -0,0 +1,64 @@
+{ lib, stdenv, fetchurl, unzip, makeDesktopItem, nwjs084, wrapGAppsHook3, gsettings-desktop-schemas, gtk3 }:
+
+let
+  nwjs = nwjs084;
+  pname = "blheli-configurator";
+  desktopItem = makeDesktopItem {
+    name = pname;
+    exec = pname;
+    icon = pname;
+    comment = "BLHeli configuration tool";
+    desktopName = "BLHeli Configurator";
+    genericName = "BLHeli ESC configuration tool";
+  };
+in
+stdenv.mkDerivation rec {
+  inherit pname;
+  version = "1.2.0";
+  src = fetchurl {
+    url = "https://github.com/blheli-configurator/blheli-configurator/releases/download/${version}/BLHeli-Configurator_linux64_${version}.zip";
+    sha256 = "1lyyxjf2ql818m59l45nsa4pwqxgwfrlzsg7209r1n5g2f8h36va";
+  };
+
+  icon = fetchurl {
+    url = "https://raw.githubusercontent.com/blheli-configurator/blheli-configurator/1.2.0/images/icon_128.png";
+    sha256 = "10miji4q2zlyql65nzq4f643x9b8z1bn9paf42vdpx1w0ma6100g";
+  };
```

Comment:
This should not be an attribute of `mkDerivation`. Maybe move it into `let` block.